src/components/books/epub-reader.tsx

```diff
@@ -97,7 +97,7 @@ export function EpubReader(props: EpubReaderProps) {
   useEffect(() => {
     if (renditionRef.current) {
       renditionRef.current.themes.override("background", theme === "dark" ? "#050505" : "#fff");
-      renditionRef.current.themes.override("color", theme === "dark" ? "#fff" : "#000");
+      renditionRef.current.themes.override("color", theme === "dark" ? "#fff" : "#050505");
```

Comment:
Shouldn't the other usages of #000 in the styling config also change into this if this should be changed?